Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -189,6 +200,33 @@ extension CommentRepliesViewController: CommentComposerViewDelegate {
   }
 }
 
+// MARK: - CommentCellDelegate
+
+extension CommentRepliesViewController: CommentCellDelegate {
+  func commentCellDidTapHeader(author _: Comment.Author) {
+    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
+
+    if #available(iOS 15.0, *) {
+      if let sheet = self.view.filter({ $0.tag == actionSheetTag }) {
+        sheet.removeFromSuperview()
+      }
+
+      let actionSheet =
+        UIHostingController(rootView: BlockUserActionSheetView(
+          blockUser: { self.blockUser() }
```

Comment:
nit: `self.blockUser` instead of closure + calling function

Reply:
Normally yes, but with this SwiftUI view, we need this because `BlockUserActionSheetView` expects `blockUser` to be of type `() -> Void` rather than `()`, which is what self.blockUser is.